Added Nvidia Control Panel Settings

Sirbucket opened the pull request:
A great guide on Nvidia control panel settings added, many of the settings are shared with Radeon so the guide can be used for AMD GPUs as well.

Sirbucket committed:
Added Nvidia Control Panel Settings

A great guide on Nvidia control panel settings added, many of the settings are shared with Radeon so the guide can be used for AMD GPUs as well.

Sirbucket committed:
Update windows.md

Sirbucket committed:
Update windows.md

Sirbucket committed:
Update windows.md

mastercoms commented on docs/os/windows.md:
```diff
@@ -64,7 +64,7 @@ Use [TCP Optimizer](https://www.speedguide.net/downloads.php) to optimize your i
 Big idea: disable as many overlays as possible. They interrupt the rendering process with their own, which can severely affect performance in many causes. Common examples of overlays are: Discord and Nahimic.
 
-### Disabling Nahimic
+## Disabling Nahimic
```
Nahimic is an overlay, so it should be listed under disable overlays.

Sirbucket replied:
I did that so it fit in uniform with the other ones, all the others had 2 ## so I thought 3 looked weird.

Sirbucket replied:
Is the third # supposed to mean it's a new section then?

mastercoms replied:
Yes the third means it is a 3rd level header, thereby making a section under the 2nd level header above.

mastercoms commented on docs/os/windows.md:
```diff
@@ -86,3 +86,9 @@ Badly programmed kernel-mode device drivers might cause latency issues, which le
+
+## Nvidia Control Panel Settings (Can be translated to AMD Radeon Settings)
+
+[Nvidia Control Panel Optimization](https://www.youtube.com/watch?v=ugadZ86uTik) Does an excellent job explaining how to configure the Nvidia control panel and goes over each setting in depth and why to use it/not use it. Many settings in the Nvidia control panel are shared with the Radeon panel so you can go off of this guide for it as well.
```
This is meant to be a text guide, with additional sources for users to expand on the reasoning behind a recommendation or to better understand it. It shouldn't be a way to list a bunch of other tutorials, especially YouTube videos.

Sirbucket replied:
Ahh understood, did you see my one on power though (It has a tutorial video but I can remove that one if you don't want and I can summarize all the settings into text for this part.)

Sirbucket replied:
Saw what you said, a OK, I'll rework this though to remove the tutorial like format from youtube videos and make a simple google doc detailing optimal settings for performance, middle ground, and visuals

micwoj92 replied:
why google doc?

Sirbucket replied:
It's simple to put it in there and put in a link, I can just figure out how to upload a text file link to github if that's what you want me to do?

Sirbucket replied:
If you don't like the specific settings lists I can delete those and just have the details document

micwoj92 replied:
Why do link to google docs when this is mastercomfig **docs**?

Sirbucket replied:
I thought it'd be too cluttered, you'd need to add an entirely seperate section if you wanted me to include it in the mastercomfigs doc as a standalone.

Sirbucket replied:
Well I mean I could do it myself but it'd be a stretch to add, easier to just shrink it down into 4 links

Sirbucket committed:
Settings rework & Title uniformity

Added non video format details put into a neat list and capitalized certain parts of titles to be more uniform.

Sirbucket reviewed (commented):
Also I do understand that both of those videos are pretty offensive so I completely get why you wouldn't want that in there anyways, I just liked them because the dude took time to look into the settings.

mastercoms commented on docs/os/windows.md:
```diff
@@ -79,10 +79,24 @@ Nahimic is an audio effects software preinstalled on some gaming laptops and des
+
+Here will be listed a few documents detailing optimal settings.
+
+[DESCRIPTION](https://docs.google.com/document/d/1-k14uBdWv3RDIfdfvR6ICCA9wYgUYjflgd-NJkIFIiI/edit?usp=sharing): Describes each setting in a general overview
```
External links are not allowed in this fashion. Such a list of customizations should be tracked and reviewed in the docs.

Sirbucket replied:
Got it, I'll figure out a way to put it in without cluttering the page and I'll remove the change to the #

Tiagoquix commented on docs/os/windows.md:
```diff
@@ -4,7 +4,7 @@ description: Learn how to optimize your Windows OS for better performance.
 # Windows
 
-## Enable Hardware-accelerated GPU scheduling
+## Enable Hardware-accelerated GPU Scheduling
```
Since you are capitalizing the titles, I think you should put the letter A from "accelerated" as a capital letter as well.

mastercoms commented:
Closing due to inactivity.
